hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoCreateSuite.java

```diff
@@ -104,66 +104,60 @@ public class CryptoCreateSuite {
-                                .balance(ONE_HUNDRED_HBARS)
-                                .declinedReward(true)
-                                .stakedNodeId(0),
-                        getAccountInfo("civilianWORewardStakingNode")
-                                .has(accountWith()
-                                        .isDeclinedReward(true)
-                                        .noStakedAccountId()
-                                        .stakedNodeId(0)))
-                .when(
-                        cryptoCreate("civilianWORewardStakingAcc")
-                                .balance(ONE_HUNDRED_HBARS)
-                                .declinedReward(true)
-                                .stakedAccountId(ACCOUNT_ID),
-                        getAccountInfo("civilianWORewardStakingAcc")
-                                .has(accountWith()
-                                        .isDeclinedReward(true)
-                                        .noStakingNodeId()
-                                        .stakedAccountId(ACCOUNT_ID)))
-                .then(
-                        cryptoCreate("civilianWRewardStakingNode")
-                                .balance(ONE_HUNDRED_HBARS)
-                                .declinedReward(false)
-                                .stakedNodeId(0),
-                        getAccountInfo("civilianWRewardStakingNode")
-                                .has(accountWith()
-                                        .isDeclinedReward(false)
-                                        .noStakedAccountId()
-                                        .stakedNodeId(0)),
-                        cryptoCreate("civilianWRewardStakingAcc")
-                                .balance(ONE_HUNDRED_HBARS)
-                                .declinedReward(false)
-                                .stakedAccountId(ACCOUNT_ID),
-                        getAccountInfo("civilianWRewardStakingAcc")
-                                .has(accountWith()
-                                        .isDeclinedReward(false)
-                                        .noStakingNodeId()
-                                        .stakedAccountId(ACCOUNT_ID)),
-                        /* --- sentiel values throw */
-                        cryptoCreate("invalidStakedAccount")
-                                .balance(ONE_HUNDRED_HBARS)
-                                .declinedReward(false)
-                                .stakedAccountId("0.0.0")
-                                .hasPrecheck(INVALID_STAKING_ID),
-                        cryptoCreate("invalidStakedNode")
-                                .balance(ONE_HUNDRED_HBARS)
-                                .declinedReward(false)
-                                .stakedNodeId(-1L)
-                                .hasPrecheck(INVALID_STAKING_ID));
+        return hapiTest(
+                cryptoCreate("civilianWORewardStakingNode")
```

Comment:
Consider refactoring this test to have less than 7 SpecOperation calls (possibly using `HapiSuite.flattened()` )